qemu/tests/qemu_img_supports_coroutines_complete.py

```diff
@@ -0,0 +1,65 @@
+import os
+import logging
+
+from avocado.utils import process
+from virttest import data_dir, utils_package
+from virttest.qemu_io import QemuIOSystem
+from virttest.qemu_storage import QemuImg
+
+
+def run(test, params, env):
+    """
+    qemu-img supports coroutines complete.
+
+    1. create source and target images, write 0 into
+       the entire source image through qemu-io
```

Comment:
entire? From the log file, I noticed the virtual size of the tested image is 20G. We just wrote 1G data to it.